src/screens/Key/Create/CreateKeyStep1/index.js

```diff
@@ -1,62 +1,59 @@
 import React, { Component } from 'react';
 import {
   View,
-  Text,
-  Image,
-  StatusBar,
-  TouchableOpacity
 } from 'react-native';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { Navigation } from 'react-native-navigation';
-import Images from '../../../../global/AssetsImages';
+
 import styles from './styles';
 import Colors from '../../../../global/Colors';
-import Header from '../../../../components/Header';
-import Screens from '../../../../global/Screens';
+import { screen } from '../../../../global/Screens';
 import Button from '../../../../components/common/Button';
+import BackgroundImage from '../../../../components/common/BackgroundImage';
+import Text from '../../../../components/common/Text';
+import FakeNavigationBar from '../../../../components/common/FakeNavigationBar';
+import KeyBaseScreen from '../../KeyBaseScreen';
+import { removePrivateKey } from '../../../../actions/key';
 
-class CreateKeyStep1 extends Component {
+class CreateKeyStep1 extends KeyBaseScreen {
 
   onNextButtonPressed() {
-    this.props.navigator.push(Screens.CREATE_KEY_SCREEN_STEP_2);
-  }
-
-  onCancelButtonPressed() {
-    alert('cancel');
+    this.props.navigator.push(screen('CREATE_KEY_SCREEN_STEP_2'));

```

Comment:
Can we rename 'CREATE_KEY_SCREEN_STEP_2' to something that makes more sense?

Reply:
I mean something that is more meaningful, like "CREATE_KEY_SCREEN_VERIFICATION_PART_ONE"

Reply:
You're right, done.